src/frontend/apps/impress/src/features/header/components/Header.tsx

```diff
@@ -1,89 +1,85 @@
+      $css={css`
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: space-between;
+        height: ${HEADER_HEIGHT}px;
+        min-height: ${HEADER_HEIGHT}px;
+        padding: 0 ${spacings['base']};
+        background-color: ${colors['greyscale-000']};
+        border-bottom: 1px solid ${colors['greyscale-200']};
+      `}
     >
-      <Box
-        $margin={{
-          left: 'big',
-          right: isSmallMobile ? 'none' : 'big',
-        }}
-        $align="center"
-        $justify="space-between"
-        $direction="row"
-      >
-        <Box>
-          <StyledLink href="/">
-            <Box
-              $align="center"
-              $gap="0.8rem"
-              $direction="row"
-              $position="relative"
-              $height="fit-content"
-              $margin={{ top: 'auto' }}
-            >
-              <Image priority src={IconDocs} alt={t('Docs Logo')} width={25} />
-              <Text
-                $padding="2px 3px"
-                $size="8px"
-                $background="#368bd6"
-                $color="white"
-                $position="absolute"
-                $radius="5px"
-                $css={`
-                  bottom: 13px;
-                  right: -17px;
-                `}
-              >
-                BETA
-              </Text>
-              <Text
-                $margin="none"
-                as="h2"
-                $color="#000091"
-                $zIndex={1}
-                $size="1.30rem"
-                $css="font-family: 'Marianne'"
-              >
-                {t('Docs')}
-              </Text>
-            </Box>
-          </StyledLink>
+      {!isDesktop && (
+        <Button
+          size="medium"
+          onClick={toggleMobileMenu}
+          aria-label={t('Open the header menu')}
+          color="primary-text"
+          icon={<Icon iconName="menu" />}
+        />
+      )}
+
+      <StyledLink href="/">
+        <Box
+          $align="center"
+          $gap={spacings['3xs']}
+          $direction="row"
+          $position="relative"
+          $height="fit-content"
+          $margin={{ top: 'auto' }}
+        >
+          <Image priority src={IconDocs} alt={t('Docs Logo')} width={25} />
+
+          <Text
+            $margin="none"
+            as="h2"
+            $color="#000091"
+            $zIndex={1}
+            $size="1.30rem"
+          >
+            {t('Docs')}
+          </Text>
+        </Box>
+      </StyledLink>
+      {!isDesktop ? (
+        <Box $direction="row" $gap={spacings['sm']}>
+          <LaGaufre />
+        </Box>
+      ) : (
+        <Box $align="center" $gap={spacings['sm']} $direction="row">
+          <ButtonLogin />
+          <LanguagePicker />
+          <LaGaufre />
         </Box>
-        {isSmallMobile ? (
-          <Box $direction="row" $gap="2rem">
-            <LaGaufre />
-            <DropdownMenu />
```

Comment:
If you don't use DropdownMenu, we should remove it from the repo (own commit as well, it would be easy if we want to get it back).